ios/RNSentryBreadcrumb.m

```diff
@@ -0,0 +1,33 @@
+#import "RNSentryBreadcrumb.h"
+@import Sentry;
+
+@implementation RNSentryBreadcrumb
+
++(SentryBreadcrumb*) from: (NSDictionary *) dict
+{
+    SentryBreadcrumb* crumb = [[SentryBreadcrumb alloc] init];
+
+    NSString * levelString = dict[@"level"];
+    SentryLevel sentryLevel;
+    if ([levelString isEqualToString:@"fatal"]) {
+        sentryLevel = kSentryLevelFatal;
```

Comment:
hello @krystofwoldrich, 

congrats on the 5.23.0 release tho.

i've just installed it and i'm having some errors due to these variables not being defined.

Reply:
Same problem for me

Reply:
There's now an issue to track this issue with the undeclared identifiers: #3863